gloo/docs/operator_guide/observability.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Observability
+weight: 3
+---
+
+## Observability
```

Comment:
We could call this `Observability (Enterprise)`, and also not sure we need the `## Observability` heading here

Reply:
It's not really enterprise though, I outline the stuff Open source Gloo ships with

Reply:
I'll remove the header though